frontend/src/components/Chat.tsx

```diff
@@ -190,6 +204,10 @@ const Chat = () => {
     setIsCreatingQuiz(!isCreatingQuiz);
   };
 
+  const handleReaction = (i: number) => {
+    socket.emit('reaction', i);
```

Comment:
nigdy za wiele deskryptywnych zmiennych :sun_with_face: 
```suggestion
  const handleReaction = (reactionId: number) => {
    socket.emit('reaction', reactionId);
```

Reply:
to "i" było mocno przypałowe - zmieniłem